e2e/lib/k8up.bash

```diff
@@ -29,7 +36,7 @@ kustomize() {
 }
 
 restic() {
-	kubectl run wrestic \
+	kubectl run "wrestic-$(randstr 5)" \
```

Comment:
why the randomized suffix?

Reply:
Because it seams that `kubectl` returns befor the Pod it creates is actually removed. So, when instantly after `kubectl` returns a new on-demand Pod with the same name is created, then it fails with a `Pod already exists` error.

Reply:
Since `$(</dev/urandom)` seems to not work in GH Actions, I've had to change this slightly to rely on timestamps instead of a random string. I've also added some explanation to the commit that introduces these suffixes.

Reply:
Have you tried the `--wait` flag?
> If true, wait for resources to be gone before returning. This waits for finalizers.